### PR TITLE
Update readme with new onboarding steps and status badges

### PR DESCRIPTION
**Pull request by ntsummers1**
## Description
Fixes [MDCT-323](https://qmacbis.atlassian.net/browse/MDCT-323). Provides a new and improved readme experiences to tingle your senses and make setup a breeze!

### How to test
Readme.

## Code author checklist
- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research) tests
- [x] I have added analytics, if necessary
- [x] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,49 @@
...
+Project Goals:
 
-### Node
+- Improve monitoring and oversight of managed care as the dominant delivery system for medicaid/CHIP.
+- Generate and analyze state-specific and nationwide data across the universe of managed care programs and requirements.
+- Identify and target efforts to assist states in improving their managed care programs.
+- Ensure compliance with managed care statutes and regulations, such as ensuring access to care.
 
-We enforce using a specific version of node, specified in the file `.nvmrc`. This version matches the Lambda runtime. We recommend managing node versions using [NVM](https://github.com/nvm-sh/nvm#installing-and-updating).
+## Table of contents
```
Comment (braxex) [inline README.md:17]:
```suggestion
## Table of Contents
```

File: README.md [hunk 2]
```diff
@@ -85,31 +81,29 @@ S3_ATTACHMENTS_BUCKET_NAME=uploads-main-attachments-446712541566
...
 ```
 
-## Running the database locally
+### Logging in
 
-In order to run dynamodb locally you will need to have java installed on your system. If not currently installed, [download the latest version](https://java.com/en/download/).
+(Make sure you've finished setting up the project locally above before moving on to this step!)
 
-If you want to a visual view of your dynamodb after the application is up and running you can install [the dynamodb-admin tool](https://www.npmjs.com/package/dynamodb-admin).
+Once you've run `./dev local` you'll find yourself on a login page at localhost:3000. For local development there is a list of users that can be found at services/ui-auth/libs/users.json. Thats where you can grab an email to fill in.
```
Comment (braxex) [inline README.md:88]:
```suggestion
Once you've run `./dev local` you'll find yourself on a login page at localhost:3000. For local development there is a list of users that can be found at services/ui-auth/libs/users.json. That's where you can grab an email to fill in.
```

File: README.md [hunk 3]
```diff
@@ -138,33 +132,66 @@ Unit tests can use [jest-axe](https://github.com/nickcolley/jest-axe), [pa11y](h
...
 Integration tests can use [cypress-axe](https://github.com/component-driven/cypress-axe) and [cypress-audit/pa11y](https://mfrachet.github.io/cypress-audit/guides/pa11y/installation.html).
 
-### Formatting
+### Prettier Linter
 
-We use Prettier to format all code. This runs as part of a Git Hook and changes to files will cause the deploy to fail.
+We use Prettier to format all code. This runs as part of a Git Hook and changes to files will cause the deploy to fail. If you followed the instructions above this is already installed and configured.
 
 Most IDEs have a Prettier plugin that can be configured to run on file save. You can also run the format check manually from the IDE or invoking Prettier on the command line.
+!!!! Is this right?
```
Comment (braxex) [inline README.md:140]:
```suggestion
```

**Review by braxex — COMMENTED** [review]
I'm fine with all of this in general, but would like Dan to do a once-over for tone and content. 👍

File: README.md [hunk 4]
```diff
@@ -1,53 +1,49 @@
-# mdct-mcr
+# MDCT-MCR
 
-A serverless form submission application built and deployed to AWS with the Serverless Application Framework.
+[![Test Coverage](https://api.codeclimate.com/v1/badges/0e158d201ebb0e226139/test_coverage)](https://codeclimate.com/github/CMSgov/mdct-mcr/test_coverage)
+[![Maintainability](https://api.codeclimate.com/v1/badges/0e158d201ebb0e226139/maintainability)](https://codeclimate.com/github/CMSgov/mdct-mcr/maintainability)
```
[the remark this answers is not in the document]
Comment (ntsummers1) [reply README.md]: Swapped!

**Review by gmrabian — APPROVED** [review]
This is great! Thanks for filling this out!

**Review by karla-vm — APPROVED** [review]
❤️

**Review by BearHanded — APPROVED** [review]

**Pull request merged**